platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/shadowgraph/Shadowgraph.java

```diff
@@ -145,7 +143,7 @@ public Shadowgraph(
      *
      * @param eventWindow the starting event window
      */
-    public synchronized void startWithEventWindow(@NonNull final EventWindow eventWindow) {
+    private void startWithEventWindow(@NonNull final EventWindow eventWindow) {
```

Comment:
why remove `synchronized`?

Reply:
This method is called from exactly one place which is synchronized.

```
public synchronized void updateEventWindow(@NonNull final EventWindow eventWindow) {
        if (this.eventWindow == null) {
            startWithEventWindow(eventWindow);
            return;
        }
```